src/lib/util.ts

```diff
@@ -247,6 +247,7 @@ export interface Quote {
 	id: number
 	quote: string
 	url?: string
+	attachmentURL?: string | undefined
```

Comment:
```suggestion
	attachmentURL?: string
```

Reply:
This breaks the type checking in one place, but i can change it

Reply:
Like I said above we can’t upload undefined values to the db, so that should be fixed instead